test/parallel/test-eslint-documented-errors.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+require('../common');
+
+const RuleTester = require('../../tools/eslint').RuleTester;
+const rule = require('../../tools/eslint-rules/documented-errors');
+
+const invalidCode = 'UNDOCUMENTED ERROR CODE';
+
+new RuleTester().run('documented-errors', rule, {
```

Comment:
I could be missing something, but do we need to have this test file if we already have the lint rule, and it's enabled for `lib/internal/errors.js`? In theory the lint rule should run as part of `make lint`, right? 😬

Reply:
The test file tests the lint rule -- `make lint` runs it.

Reply:
This test file is intended to test the lint rule itself. For example, if the rule had a bug where it never reported any errors, we probably wouldn't notice it when running `make lint`.

Reply:
👍 sorry, my bad, I read `RuleTester` as running the rule.